services/ai-ml/gen-ai/features.yaml

```diff
@@ -0,0 +1,145 @@
+common_features:
+  - CCC.F01 # Encryption in transit
+  - CCC.F02 # Encryption at rest
+  - CCC.F03 # Access/Activity Logs
+  - CCC.F06 # Identity Based Access Control
+  - CCC.F09 # Monitoring
+  - CCC.F10 # Logging
+  - CCC.F13 # Infrastructure as Code
+  - CCC.F14 # API Access
+  - CCC.F15 # Cost Management
+  - CCC.F16 # Budgeting
+  - CCC.F18 # Versioning
+  - CCC.F19 # On Demand Scaling
+  - CCC.F20 # Tagging
+features:
+  - id: CCC.GenAI.F01
+    title: Access for text base models
+    description: |
+      Ability to access foundation models that excel at natural language 
+      understanding and generation tasks such as summarization, translation, 
+      content generation, and more.
```

Comment:
Do we need the "and more" bit? If we are defining coming features this leaves some room for ambiguity?